docs/src/search.md

```diff
@@ -138,6 +138,36 @@ LanceDB supports binary vectors as a data type, and has the ability to search bi
         --8<-- "python/python/tests/docs/test_binary_vector.py:async_binary_vector"
         ```
 
+## Multivector type
+
+LanceDB supports multivector type, this is useful when you have multiple vectors for a single item (e.g. with ColBert and ColPali).
```

Comment:
This is still missing an intuitive explanation of what this query type means, and how it's different from the single vector situation. Could you perhaps root this in an example? Is the main example chunked documents, where each row is a full document, and each vector made for a chunk?